### PR TITLE
FD-375: Cancel when delete application

### PR DESCRIPTION
**Pull request by adamthewan**

### CODE REVIEW

File: src/screens/Apps/MyApps/MyApps.tsx [hunk 1]
```diff
@@ -119,7 +122,10 @@ const MyAppsComponent: React.FunctionComponent<IMyAppsProps> = props => {
   };
   const handleDeleteConfirm = () => {
     setShowDeleteDialog(false);
-    purgeApps(indexes);
+    // tslint:disable-next-line:no-console
```
Comment (starkyru) [inline src/screens/Apps/MyApps/MyApps.tsx]: clean up after debugging :)

File: src/redux/effects/api/appsSagas.ts [hunk 2]
```diff
@@ -16,6 +16,20 @@ function* purgeAppsSaga() {
...
+function* cancelAppsSaga() {
+  while (true) {
+    try {
+      const action = yield take(AppsActionTypes.APPS_DELETE);
+      // yield put(AppsActions.setPurgeFlag(true));
+      yield all(
+        action.payload.ids.map((id: string) => put(AppsActions.cancelApp(id))),
+      );
+    } finally {
+      // yield put(AppsActions.setPurgeFlag(false));
```
Comment (starkyru) [inline src/redux/effects/api/appsSagas.ts]: no commented code allowed.

File: src/redux/effects/api/appsSagas.ts [hunk 3]
```diff
@@ -16,6 +16,20 @@ function* purgeAppsSaga() {
     }
   }
 }
+function* cancelAppsSaga() {
+  while (true) {
+    try {
+      const action = yield take(AppsActionTypes.APPS_DELETE);
+      // yield put(AppsActions.setPurgeFlag(true));
```
Comment (starkyru) [inline src/redux/effects/api/appsSagas.ts]: no commented code allowed.

Comment (starkyru) [reply src/redux/effects/api/appsSagas.ts]: Also, please make sure that UI shows a loading indicator while deleting apps.

Comment (adamthewan) [reply src/redux/effects/api/appsSagas.ts]: Alright sure :)

**Pull request merged**